pkg/asset/manifests/operators_test.go

```diff
@@ -131,3 +132,114 @@ sshKey: test-ssh-key
 	}
 	assert.Equal(t, expectedConfig, ic, "install config was unexpectedly modified")
 }
+
+// TestRedactedInstallConfigNutanix tests the redactedInstallConfig function for Nutanix platform.
+func TestRedactedInstallConfigNutanix(t *testing.T) {
+	createInstallConfigWithNutanix := func() *types.InstallConfig {
+		return &types.InstallConfig{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "test-nutanix-cluster",
+			},
+			SSHKey:     "test-ssh-key",
+			BaseDomain: "test-domain",
+			Networking: &types.Networking{
+				MachineNetwork: []types.MachineNetworkEntry{
+					{CIDR: *ipnet.MustParseCIDR("1.2.3.4/5")},
+				},
+				NetworkType: "test-network-type",
+				ClusterNetwork: []types.ClusterNetworkEntry{
+					{
+						CIDR:       *ipnet.MustParseCIDR("1.2.3.4/5"),
+						HostPrefix: 6,
+					},
+				},
+				ServiceNetwork: []ipnet.IPNet{*ipnet.MustParseCIDR("1.2.3.4/5")},
+			},
+			ControlPlane: &types.MachinePool{
+				Name:         "control-plane",
+				Replicas:     pointer.Int64Ptr(3),
+				Architecture: types.ArchitectureAMD64,
+			},
+			Compute: []types.MachinePool{
+				{
+					Name:         "compute",
+					Replicas:     pointer.Int64Ptr(3),
+					Architecture: types.ArchitectureAMD64,
```

Comment:
nit: Lint is complaining about `pointer.Int64Ptr` (See [lint jobs](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_installer/9931/pull-ci-openshift-installer-main-golint/1965432927772741632)). Interestingly, it only complains about new changes.

Let's use `ptr.To()` from ``"k8s.io/utils/ptr"``?